presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -116,6 +116,27 @@ Memory Management Properties
     This is the amount of memory set aside as headroom/buffer in the JVM heap
     for allocations that are not tracked by Presto.
 
+
+Query Management Properties
+---------------------------
+
+``query.max-history``
+^^^^^^^^^^^^^^^^^^^^^
+    * **Type:** ``integer``
+    * **Default value:** ``100``
+
+    The maximum amount of queries to keep in the query history to access
+    statistics and other information about.
+
+``query.min-expire-age``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``duration``
+    * **Default value:** ``15 min``
+
+    The minimal age of a query before it is expired. The supported, but legacy
+    property name is ``query.max-age``.
```

Comment:
No need to document legacy property name IMO.

Reply:
Also, would be good to document interaction between max-history and min-expiry-age -- which "wins"

Reply:
its min-expire-age btw..

Reply:
And in terms of interaction ... dont know. I am guessing that queries that reach their expire age get removed from history no matter if it is full or not..  and once buffer is full even younger queries queries get killed based on age. But thats just a guess... so .. is this correct? And is it really worth documenting?

Reply:
I added some more description for each so things should be clear imho.